submissions/onyinyeChukwuka/css/sign-up.css

```diff
@@ -0,0 +1,247 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    background-color: #ffffff;
+}
+
+.header {
+    justify-content: space-between;
+    padding: 2px;
+    background-color: #0066f5;
+}
+
+nav {
+    padding: 20px;
+}
+
+.left-side {
+    display: flex;
+}
+
+nav>.right-side>div {
+    margin-right: 100px;
+    font-size: 16px;
+    color: #ffffff;
+    line-height: 1.19;
+    text-align: left;
+}
+
+.tab {
+    color: #ffffff;
+    text-decoration: none;
+}
+
+.link-wrapper {
+    height: 22px;
+}
+
+.link-wrapper a {
+    color: #ffffff;
+    text-decoration: none;
+}
+
+.sidebar h4 {
+    font-size: 50px;
+    font-weight: bold;
+    text-align: left;
+    line-height: 1.22;
+    color: #0066f5;
+    margin-bottom: 23px;
+    margin-top: 175px;
+}
+
+.sidebar p {
+    font-family: Montserrat;
+    font-size: 30px;
+    text-align: left;
+    line-height: 1.23;
+    color: #5f5f5f;
+}
+
+.sidebar strong {
+    color: #0066f5;
+    text-decoration: underline;
+    text-underline-position: below;
+}
+
+form {
+    font-family: Montserrat;
+    font-size: 18px;
+    line-height: 1.22;
+    text-align: left;
+    color: #5f5f5f;
+    margin-top: 40px;
+}
+
+.form {
+    border-radius: 23px;
+    border: solid 1px #707070;
+    background-color: #ffffff;
+    margin-top: 46px;
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-bottom: 20px;
+}
+
+.presentation {
+    width: 90%;
+    margin: auto;
+    margin-bottom: 180px;
+}
+
+.sidebar {
+    flex: 1;
+    margin-right: 10px;
+}
+
+.texts{
+    /* width: 521px; */
+    width: 100%;
+    height: 50px;
+    border-radius: 10px;
+    border: solid 2px #5f5f5f;
+    margin-bottom: 5px;
+}
+
+#terms {
+    font-family: Montserrat;
+    font-size: 40px;
+    font-weight: bold;
+    line-height: 1.22;
+    text-align: left;
+    color: #0066f5;
+}
+
+#signupbutton {
+    width: 196px;
+    height: 48px;
+    border-radius: 4px;
+    background-color: #0066f5;
+    font-family: Raleway;
+    font-size: 18px;
+    font-weight: bold;
+    line-height: 0.67;
+    text-align: center;
+    color: #ffffff;
+}
+
+.confirm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#submitbutton {
+    width: 108px;
+    height: 44px;
+    border: solid 1px #707070;
+    background-color: #ffffff;
+    font-family: Montserrat;
```

Comment:
@Onyinye101 Remove  the font family here. You have declared it in the body tag already which cuts across all the html elements within it.